partials/language-specific-deploy/php.md

```diff
@@ -36,15 +36,16 @@ of the configuration file *clevercloud/php.json* with the absolute path (*from t
 
 In the following example we want to set the webroot to the folder `/public`:
 
-```javascript
-  {
+```json
+{
     "deploy": {
-      "webroot": "/public"
+        "webroot": "/public"
     }
-  }
+}
 ```
+
 {{< alert "warning" "Warning:" >}}
-    <p>Please note the absolute path style: `/public`. The change of the webroot will be rejected during the deployment if the target directory does not exist or is not a directory.</p>
+    Please note the absolute path style: `/public`. The change of the webroot will be rejected during the deployment if the target directory does not exist or is not a directory.
```

Comment:
The backticks here won't be rendered

Reply:
On my side it is rendered, I use `hugo v0.83.1`.

Reply:
Mh, that's not the version we currently use but we can update. Did you notice any issues with this version?

Reply:
I'll upgrade the hugo version, it's indeed okay with the last one